src/opcodes/tests/test_stack.cairo

```diff
@@ -291,6 +291,17 @@ fn test_op_2rot_insufficient_items() {
     utils::check_dstack_size(ref engine, 5);
 }
 
+#[test]
+fn test_max_stack_size() {
```

Comment:
Could you add a test that adds 1000 elements and doesn't throw an error?

Reply:
ok